evaluation.html

```diff
@@ -0,0 +1,68 @@
+<DOCTYPE html>
+    <html>
+        <head><title></title>
+
+            <link rel="stylesheet" type="text/css" href="css/evaluation.css">
+        </head>
+
+        <body>
+
+            <header>
+                <a id="homeLogo" href="login.html"><img src="assets/Logo.png"></a>   
+                <a class="pgSwitch borderSwitch" href="evaluation.html">Evaluations</a>
+<!--                @Alex there are no restrictions on number of classes-->
+                <a class="pgSwitch" href="New%20evaluation.html">New Evaluation</a>
+                <button id="Logout" type="button", Value="Logout", name="Logout">Logout</button>
+
+
+            </header>
+
+            <table align="center">
+                <colgroup>
+<!--                    @Alex the <col> in this case sets the width of each column in the table in raport to the total table width, it helps me stylize it . I could have also used <col> span but I don't think it would have been semantically correct-->
```

Comment:
Not particularly sure what you mean when you say `<col> span`, but your usage sounds reasonable otherwise. You're good to go. ;)